src/setuptools_scm/__init__.py

```diff
@@ -116,12 +121,12 @@ def _do_parse(config):
 
 def get_version(
     root=".",
-    version_scheme="guess-next-dev",
-    local_scheme="node-and-date",
+    version_scheme=DEFAULT_VERSION_SCHEME,
```

Comment:
unfortunately the argument order is different compared to Configuration
(this is a general oversight)

i do wonder if perhaps a breaking release should be done about this and the arguments (with some more change accumulated on top)

Reply:
Is the constructor for `Configuration` part of a public interface? I'd have imagined that was private, so could adapt to match `get_version` (maybe; I haven't tried) without affecting compatibility.

Reply:
In setuptools_scm I have made some mistakes about the semantics of public vs private, it may be necessary to take time for a detailed workout on that

Reply:
No problem. Happy to proceed with it as is.